document/core/util/bikeshed_fixup.py

```diff
@@ -41,13 +41,14 @@ def Main():
     <div class="related" role="navigation" aria-label="related navigation">
       <h3>Navigation</h3>
       <ul>
-        <li class="nav-item nav-item-0"><a href="index.html#document-index">WebAssembly 1.0</a> &#187;</li> 
+        <li class="nav-item nav-item-0"><a href="#">WebAssembly 1.0</a> &#187;</li>
+        <li class="nav-item nav-item-this"><a href="">WebAssembly 1.0</a></li> 
```

Comment:
While you're here, could you change this to 1.1, to match the version given on the regular front page?

Reply:
Done!